assets/js/custom/TranslateComments.js

```diff
@@ -14,7 +14,12 @@ export class TranslateComments extends Translation {
       const commentId = $(this)
         .attr('id')
         .substring('comment-translation-button-'.length)
-      const translateCommentUrl = $('.comment-translation').data(
+      const matchingContainer = $(
+        '.comment-translation[data-translate-comment-id="translate-comment-' +
```

Comment:
please try to use only "vanilla" javascript, we are trying to get rid of jquery in the future.
I see why you did it here